rendering.ts

```diff
@@ -9,23 +9,6 @@ import { GameEngine } from './game/engine';
 import { isLootSelectionImpossible, isRoomSelectionImpossible } from './game/utils';
 import { RunEndedScreen } from './components/RunEndedScreen';
 
-const getLoadingText = (state: GameState) => {
-    if (!state) return t('global.initializing');
-    switch (state.phase) {
-        case 'AWAITING_ADVENTURER_CHOICE':
-            return t('main.adventurer_considering_offer');
-        case 'AWAITING_ENCOUNTER_FEEDBACK':
-            return t('main.adventurer_facing_encounter');
-        default:
-            return t('global.loading');
-    }
-};
-
-const renderLoadingIndicator = (state: GameState) => {
-    const loadingIndicator = document.createElement('loading-indicator');
```

Comment:
If the LoadingIndicator is no longer used you should remove it as well (but please check that it's the case)

Reply:
You should then remove it's import from main.ts and the reference in READLE.md